### DISCUSSION TITLE
Dear Shroud Email founder - Privacy concern

### DISCUSSION THREAD

**Discussion by Bammede** [Shroud-email/shroud.email#61] [General]
Before I purchase the service will you improve privacy? Found this on appsumo:

_Shroud Email cons:

- service saves emails marked as spam. If email address gets spam, then alias can be deleted by the user. Alias deletion prevents spam. Competitors do not store any emails, because it is better for privacy.

- If you respond to reverse alias then the recipient knows you use Shroud.email (text: via Shroud.email). Less private solution.

- the service adds additional its own text to email content like "Shroud.email didn't find any trackers."

- How user data are encrypted?_

**Comment by taobojlen** [comment]
Hey, thanks for your questions! Let me respond to your points:

- Shroud.email does indeed save spam emails instead of forwarding them. The reason for this is not to prevent spam for users -- you're right that you could just delete the alias. The reason is to ensure that our mailserver maintains a high reputation. If we forward spam emails, then recipient mailservers will consider us as someone that sends spam, and this will harm the deliverability of all emails! Some of our competitors like SimpleLogin do the same.
- It's true that our outgoing emails contain the Shroud.email text. We're working on making this configurable: https://github.com/Shroud-email/shroud.email/issues/29
- We do not add the "Shroud.email didn't find any trackers" text to outgoing emails, only incoming ones. Without this, it wouldn't be possible to see which trackers we removed.
- User data is encrypted at rest, and in transit via TLS.

**Comment by Bammede** [comment]
Hey, thank you for clarification.

In my opinion adding "Shroud.email didn't find any trackers" to incoming emails can be annoying for some users (including me). If there are no trackers, no notification is better UX. If the message includes trackers, then Shroud notification is reasonable.

That's good user data are encrypted. However privacy related businesses usually brags about encryption and publish more details about it.

Overall your service is really promising.
